scripts/loki_transform.py

```diff
@@ -199,7 +201,7 @@ def convert(
 
     # Now we instantiate our transformation pipeline and apply the main changes
     transformation = None
-    if mode in ['idem', 'idem-stack']:
+    if mode in ['idem', 'idem-stack', "c"]:
```

Comment:
Why is this required? `IdemTransformation` is a do-nothing, so should not be necessary for the F2C transpilation.